src/helm/benchmark/scenarios/cleva_scenario.py

```diff
@@ -346,3 +346,38 @@ class CLEVAFactCheckingScenario(CLEVAScenario):
 
     description = "Fact checking task in CLEVA benchmark"
     tags = ["fact_checking"]
```

Comment:
Add a "harms" tag here and the comment "The disinformation task of CLEVA benchmark." should be modified to "The fact-checking task of CLEVA benchmark."